setup.py

```diff
@@ -1,8 +1,47 @@
 # -*- coding: utf-8 -*-
 
 from setuptools import setup
+from pathlib import Path
+import shutil
 import versioneer
 
+
+def copy_mda_assets():
+    static_dir = Path('.') / 'mdanalysis_sphinx_theme' / 'static'
+    branding_dir = Path('.') / 'branding'
+
+    # outdir for logos and icons
+    static_logos = static_dir / 'logo'
+
+    # main logo to appear in nav bar
+    mda_logo: Path = branding_dir / 'logos' /\
+                                    'rastered' /\
+                                    'mdanalysis-logo_bgwhite@600ppi.png'
+
+    mda_favicon = branding_dir / 'logos' /\
+                                 'icons' /\
+                                 'mdanalysis-logo.ico'
```

Comment:
Here as well -- the full thing on one line is only 73 characters, could we use that instead?